src/tests/auth.setup.js

```diff
@@ -0,0 +1,21 @@
+import { test as setup } from '@playwright/test';
+
+const authFile = 'playwright/.auth/user.json';
+
+setup('authenticate', async ({ page }) => {
+  // Perform authentication steps. Replace these actions with your own.
+  await page.goto('https://www.saucedemo.com/');
```

Comment:
Lets move `https://www.saucedemo.com/` to the baseUrl of config file

Reply:
Updated